kedro/framework/cli/starters.py

```diff
@@ -793,14 +793,15 @@ def _validate_range(start, end):
             click.secho(message, fg="red", err=True)
             sys.exit(1)
 
+    # Guard clause if tools_str is None, which can happen if prompts.yml is removed
```

Comment:
We can remove this comment, as this would now happen if tools is omitted from config - no longer a guard clause but inspected behaviour